assets/sass/components/_home-content.scss

```diff
@@ -295,3 +297,10 @@
     height: 90px;
   }
 }
+
+.govuk-hub-home-updates {
+  display: flex;
+  > div > * {
```

Comment:
This feels a bit hacky and none specific in which elements its targeting. Applying display flex to the div you have in home-new.html line #36, and applying it to the 2 child divs directly inside it achieves the same goal in a much cleaner and more readable way.

Reply:
I'd probably argue the other way is more hacky, it adds flex to two child elements with the expectation that there contents will be controlled by flex, which is not necessarily the case. The only requirement of these elements is that they take up 100% of the available height (which is a characteristic of display flex amongst others).
If its the catch all wild card nature of the selector thats jarring then I could call them out directly `> div > a, > div > div {` or just get the first childs `> div:first-child`? 🤔 
I quite like that the child styles sit with the parents `display:flex` as the proximity helps me remember they are required and related.

Reply:
🤔 Actually its not necessary at all, because both the elements in question are already set to 100% 😁

Reply:
great. yes it was the wildcard that was jarring. nice to see its not needed after all :)